packages/validator/src/services/indices.ts

```diff
@@ -110,6 +116,49 @@ export class IndicesService {
         newIndices.push(validatorState.index);
       }
     }
+
+    logValidatorStatuses(allValidators, this.logger);
+
     return newIndices;
   }
 }
+
+function logValidatorStatuses(allValidators: Map<ValidatorStatus, ValidatorResponse[]>, logger: Logger): void {
+  const loggedValidatorStates: {[key: string]: ValidatorStatus[]} = {
+    "Validator activated": ["active"],
+    "Validator pending": ["pending_initialized", "pending_queued"],
+    "Validator withdrawn": ["withdrawal_done"],
+    "Validator exited": ["exited_slashed", "exited_unslashed"],
+  };
+
+  const logFormatter = (message: string, validatorState: ValidatorResponse): void => {
```

Comment:
This function appears to be only used once and is just a single `logger.info`. Preference would be to avoid one-line functions like this unless there's a good reason (reuse of logic, etc).